finality-aleph/src/sync/forest/mod.rs

```diff
@@ -172,16 +188,18 @@ where
             vertices: HashMap::new(),
             highest_justified: top_finalized.clone(),
             justified_blocks: HashMap::new(),
+            imported_leaves: HashSet::new(),
+            favourite: top_finalized.clone(),
             root_id: top_finalized.clone(),
             root_children: HashSet::new(),
             compost_bin: HashSet::new(),
         };
 
         // Populate the forest
         let mut deque = VecDeque::from([top_finalized]);
-        while let Some(hash) = deque.pop_front() {
+        while let Some(id) = deque.pop_front() {
```

Comment:
should we also change favourite here?

Reply:
We already do, that's the advantage of using struct methods in constructors. B)

Reply:
I somehow missed tha :P